### PR TITLE
[ACM-11727]:Revise firewall reqs in bare metal docs for HCP

### PR DESCRIPTION
**Pull request by lahinson**
Related Jira: https://issues.redhat.com/browse/ACM-11727

### CODE REVIEW

**Review by swopebe — APPROVED** [review]
Beautiful!

File: clusters/hosted_control_planes/bm_intro.adoc [hunk 1]
```diff
@@ -76,7 +76,7 @@ A hosted control plane exposes the following services on bare metal:
 * `Konnectivity`
 
 ** The `Konnectivity` service runs on port 443 by default when you use the route and ingress to expose the service.
-** The `Konnectivity` agent, which establishes a reverse tunnel to allow bi-directional communication on the hosted cluster, requires egress access to the cluster API server address on port 6443. With that egress access, the agent can reach the `APIServer` service.
+** The `Konnectivity` agent establishes a reverse tunnel to allow the control plane to access the hosted cluster's network. The agent uses egress to connect to the `Konnectivity` server. The server is exposed by using either a route on port 443 or a manually assigned `NodePort`.
```
Comment (swopebe) [inline clusters/hosted_control_planes/bm_intro.adoc]: One thing, please remove possessive, per IBM Style, we want to avoid them.

Comment (lahinson) [reply clusters/hosted_control_planes/bm_intro.adoc]: Fixed

**Pull request merged**